src/layouts/BaseLayout.astro

```diff
@@ -69,7 +69,6 @@ const description = `JSHeroes is an yearly event organized by the local JS commu
 
   <body>
     <Navbar />
-    <CookieConsent />
```

Comment:
If this component isn't used anymore should we delete it?

Reply:
we can keep it just in case we need it later on